tardis/utilities/utils.py

```diff
@@ -42,3 +46,26 @@ def htcondor_csv_parser(htcondor_input, fieldnames, delimiter="\t", replacements
                 key: value if value not in replacements.keys() else replacements[value]
                 for key, value in row.items()
             }
+
+
+def slurm_cmd_option_formatter(options):
```

Comment:
Please add annotations for ``options`` here.

Reply:
Done!